### DISCUSSION TITLE
How can I start the linux version of WA2DC please

### DISCUSSION THREAD

**Discussion by wilsonlim60** [FKLC/WhatsAppToDiscord#218] [Q&A] [answered]
Hi ,
I managed to start , config WA2DC in Windows and got it running successfully.
I m trying to let it run in my VPS. I downloaded the WA2DC-Linux.
But I couldnt find any documentation about how to make it run. Is there any help/document/link ?

Thank you.

**Comment by malagoni** [comment]
if you are using the actual let's say ".exe" file, you can turn it on via terminal. 
I use "sudo node . /dev/null &" to be able to still use the terminal after the app launches, if you wish only to start it use "sudo node . "
Remember, it's the dot " . " that makes the app start.

**Reply by wilsonlim60** [reply]
Thanks man~ great help from your comment. I managed to start it using ./ by changing WA2DC-Linux file permission from 644 to 744 too.

**Reply by Stradios** [reply, marked as answer]
https://github.com/FKLC/WhatsAppToDiscord/discussions/219#discussion-7528452
